test/runtests.jl

```diff
@@ -1,128 +1,153 @@
-    @safetestset "Spectral elem - sphere gradient" begin @time include("Operators/spectralelement/sphere_gradient.jl") end
-    @safetestset "Spectral elem - sphere divergence" begin @time include("Operators/spectralelement/sphere_divergence.jl") end
-    @safetestset "Spectral elem - sphere curl" begin @time include("Operators/spectralelement/sphere_curl.jl") end
-    @safetestset "Spectral elem - sphere diffusion" begin @time include("Operators/spectralelement/sphere_diffusion.jl") end
-    @safetestset "Spectral elem - sphere diffusion vec" begin @time include("Operators/spectralelement/sphere_diffusion_vec.jl") end
-    @safetestset "Spectral elem - sphere hyperdiffusion" begin @time include("Operators/spectralelement/unit_sphere_hyperdiffusion.jl") end
-    @safetestset "Spectral elem - sphere hyperdiffusion vec" begin @time include("Operators/spectralelement/unit_sphere_hyperdiffusion_vec.jl") end
-    
-    @safetestset "FD ops - column" begin @time include("Operators/finitedifference/column.jl") end
-    @safetestset "FD ops - opt" begin @time include("Operators/finitedifference/opt.jl") end
-    @safetestset "FD ops - wfact" begin @time include("Operators/finitedifference/wfact.jl") end
-    @safetestset "FD ops - linsolve" begin @time include("Operators/finitedifference/linsolve.jl") end
-    @safetestset "FD ops - examples" begin @time include("Operators/finitedifference/opt_examples.jl") end
-    # now part of buildkite
-    # @time include("Operators/finitedifference/implicit_stencils.jl")
-    # @time include("Operators/finitedifference/opt_implicit_stencils.jl")
-    @safetestset "Hybrid - 2D" begin @time include("Operators/hybrid/unit_2d.jl") end
-    @safetestset "Hybrid - 3D" begin @time include("Operators/hybrid/unit_3d.jl") end
-    @safetestset "Hybrid - dss opt" begin @time include("Operators/hybrid/dss_opt.jl") end
-    @safetestset "Hybrid - opt" begin @time include("Operators/hybrid/opt.jl") end
-
-    @safetestset "MatrixFields - BandMatrixRow" begin @time include("MatrixFields/band_matrix_row.jl") end
-    @safetestset "MatrixFields - rmul_with_projection" begin @time include("MatrixFields/rmul_with_projection.jl") end
-    @safetestset "MatrixFields - field2arrays" begin @time include("MatrixFields/field2arrays.jl") end
-    @safetestset "MatrixFields - matrix multiplication at boundaries" begin @time include("MatrixFields/matrix_multiplication_at_boundaries.jl") end
-    @safetestset "MatrixFields - field names" begin @time include("MatrixFields/field_names.jl") end
-    # now part of buildkite
-    # @safetestset "MatrixFields - matrix field broadcasting" begin @time include("MatrixFields/matrix_field_broadcasting.jl") end
-    # @safetestset "MatrixFields - operator matrices" begin @time include("MatrixFields/operator_matrices.jl") end
-    # @safetestset "MatrixFields - field matrix solvers" begin @time include("MatrixFields/field_matrix_solvers.jl") end
+mutable struct UnitTest
```

Comment:
Why does it have to be mutable? Is it only for the `elapsed`?

Reply:
Yeah, just for `elapsed`

Reply:
I don't think it's a big deal, and it's type-stable. We can always iterate on it.